### DISCUSSION TITLE
Table filter with prefix and suffix breaks the grid of the stylesheet

### DISCUSSION THREAD

**Discussion by sasori13L** [filamentphp/filament#11514] [Help] [answered]
### Package

Table builder

### Package Version

v3.2.0

### How can we help you?

When using the filament table filter with one filter using both prefix and suffix. the grid spacing becomes too close that it overlaps the next column.

![image](https://github.com/filamentphp/filament/assets/16425810/5ed73549-013d-4668-b596-23ec359a05a1)

**Comment by leandrocfe** [comment, marked as answer]
Did you try to customize the number of columns?
https://filamentphp.com/docs/3.x/tables/filters/layout#customizing-the-filter-form-schema

**Reply by sasori13L** [reply]
Awesome! Thank you for this answer!
